mmv1/third_party/terraform/services/cloudrun/resource_cloud_run_service_test.go.erb

```diff
@@ -1441,3 +1441,71 @@ resource "google_cloud_run_service" "default" {
 }
 
 <% end -%>
+
+<% unless version == 'ga' -%>
+
+func TestAccCloudRunService_resourcesRequirements(t *testing.T) {
+	acctest.SkipIfVcr(t)
```

Comment:
Is there any particular reason that we need to skip VCR for this test?

Reply:
Thanks for catching. I have removed these tests in favor of the examples.